apps/blog/_posts/tech-post-1.md

```diff
@@ -0,0 +1,60 @@
+date: "2022-09-17"
+author:
+  name: "Rui Oliveira"
+  photo: "ruioliveira02.jpg"
+  username: "ruioliveira02"
+draft: false
+topic: "Desenvolvimento"
+featured: true
+---
+
+---
+
+&nbsp;
+
+Este é o primeiro post de uma série em que vamos discutir as plataformas do Coderdojo Braga. Como foram feitas, dificuldades técnicas, histórias engraçadas e para onde vão são alguns dos temas que podem esperar daqui. Espero sinceramente que, antes de um blog técnico, que isto seja um blog educativo, e que demonstre o trabalho necessário para fazer uma aplicação que vai ser usada por várias pessoas, e traga uma maior sensibilidade para a engenharia de software.
+
+&nbsp;
+
+## O nascimento
+
+Podem só ter sido lançadas recentemente, mas esta plataforma já se encontra em desenvolvimento há mais de um ano e meio. Se formos a ver o histórico no [Github](https://github.com/coderdojobraga/bokken), vemos que o desenvolvimento começou em março de 2021.
+
+Falando em Github, vamos começar pela premissa básica do desenvolvimento de software no CoderDojo Braga: todo o código produzido é open source. Código ser open source significa que é público e acessível por qualquer pessoa. Se quiserem, podem consultar o código [neste link](https://github.com/coderdojobraga/bokken) e [neste](https://github.com/coderdojobraga/shuriken). Além de ser público, toda a gente pode contribuir e programar para o CoderDojo Braga.
+
+Há vários exemplos de software open source que são fundamentais para o funcionamento do mundo e da internet. O mais óbvio é o sistema operativo Linux, que opera em grande maioria dos servidores na internet (incluindo aquele que hospeda este site), mas existem outros, como o Mozilla Firefox, LibreOffice, .etc.
+
+Se estão a prestar atenção ao que estão a ler, devem ter reparado que eu pus dois links diferentes para o código da plataforma. Porquê? Porque a plataforma está dividida em dois grandes componentes: o site, que estão a visitar agora - nome de código [shuriken](https://github.com/coderdojobraga/shuriken) - e o serviço responsável por toda a lógica envolvente a sessões, contas, .etc - nome de código [bokken](https://github.com/coderdojobraga/bokken).
+
+Estes serviços comunicam entre si pela internet, de tal forma que, quando fazem login na plataforma, por exemplo, o site pergunta ao servidor se as credenciais estão corretas e, se estiverem, redireciona-vos para o dashboard inicial. Todas as comunicações são encriptadas, o que garante a segurança da informação transmitida.
+
+## Desenvolvimento
+
+O site está a ser desenvolvido em [NextJS](https://nextjs.org/), e o backend (o serviço responsável pela lógica da plataforma) em [Elixir](https://elixir-lang.org/). Esta decisão acrescentou alguma complexidade ao projeto, porque obrigou a termos de tratar explicitamente da conexão entre os dois componentes da plataforma. A vantagem é que a maior separação entre o site, aquilo que vocês veem, e a lógica por trás dá uma maior liberdade a ambas as partes de
```

Comment:
Since you are linking to NextJS, a framework of JavaScript, I think you should link to Phoenix, instead of Elixir itself.